packages/fluentui/react-northstar/src/components/Menu/MenuItem.tsx

```diff
@@ -303,6 +304,15 @@ export const MenuItem = compose<'a', MenuItemProps, MenuItemStylesProps, {}, {}>
       }
     };
 
+    useEventListener({
+      target: context.target,
+      type: 'scroll',
+      listener: e => {
+        if (!isSubmenuOpen()) return;
+        trySetMenuOpen(false, (e as unknown) as React.MouseEvent);
```

Comment:
```suggestion
        trySetMenuOpen(false, e);
```

This is not `React.MouseEvent`, this is native event. Let's fix `trySetMenuOpen` to accept `MouseEvent` instead